tests/test_stdlib.py

```diff
@@ -250,3 +252,33 @@ def test_positional_args_retained(self):
             {'event': '%d %d %s', 'positional_args': positional_args})
         assert 'positional_args' in event_dict
         assert positional_args == event_dict['positional_args']
+
+
+class TestAddLogLevel(object):
+    def test_log_level_added(self):
+        """
+        The log level is added to the event dict.
+        """
+        event_dict = add_log_level(None, 'error', {})
+        assert 'level' in event_dict
```

Comment:
I've removed it. Though it will fail with a `KeyError` instead of an `AssertionError` if the test fails. But I agree, simpler is better. (Hence my combined "alias" test, btw.)